tests/functional/r/regression_02/regression_8736.py

```diff
@@ -0,0 +1,3 @@
+"""This does not crash in the functional tests, but it did when called directly."""
+
+assert "\U00010000" == "\ud800\udc00"  # [comparison-of-constants]
```

Comment:
Didn't manage to reproduce the crash automatically. Might be due to some unicode's magic already present in the functional test.